docusign/docusign.lua

```diff
@@ -0,0 +1,79 @@
+--[[
+
+# [DocuSign API DOCUMENTATION](https://docs.docusign.com/esign/guide/usage/quickstart.html) connector.
+
+### Authorization
+
+Authorization use [Basic authentication](https://admindemo.docusign.com).
```

Comment:
No, you've implemented [DocuSign legacy header authentication ](https://docs.docusign.com/esign/guide/authentication/legacy_auth.html)